src/OrchardCore.Modules/OrchardCore.Users/TimeZone/Services/UserTimeZoneService.cs

```diff
@@ -8,80 +8,77 @@
 using OrchardCore.Users.Models;
 using OrchardCore.Users.TimeZone.Models;
 
-namespace OrchardCore.Users.TimeZone.Services
+namespace OrchardCore.Users.TimeZone.Services;
+
+public class UserTimeZoneService
 {
-    public class UserTimeZoneService
+    private const string CacheKey = "UserTimeZone/";
+
+    private readonly TimeSpan _slidingExpiration = TimeSpan.FromMinutes(1);
+    private readonly IClock _clock;
+    private readonly IDistributedCache _distributedCache;
+    private readonly IHttpContextAccessor _httpContextAccessor;
+    private readonly UserManager<IUser> _userManager;
+
+    public UserTimeZoneService(
+        IClock clock,
+        IDistributedCache distributedCache,
+        IHttpContextAccessor httpContextAccessor,
+        UserManager<IUser> userManager)
     {
-        private const string CacheKey = "UserTimeZone/";
-        private readonly TimeSpan _slidingExpiration = TimeSpan.FromMinutes(1);
-
-        private readonly IClock _clock;
-        private readonly IDistributedCache _distributedCache;
-        private readonly IHttpContextAccessor _httpContextAccessor;
-        private readonly UserManager<IUser> _userManager;
-
-        public UserTimeZoneService(
-            IClock clock,
-            IDistributedCache distributedCache,
-            IHttpContextAccessor httpContextAccessor,
-            UserManager<IUser> userManager
-            )
+        _clock = clock;
+        _distributedCache = distributedCache;
+        _httpContextAccessor = httpContextAccessor;
+        _userManager = userManager;
+    }
+
+    public async Task<ITimeZone> GetUserTimeZoneAsync()
+    {
+        var currentTimeZoneId = await GetCurrentUserTimeZoneIdAsync();
+
+        if (string.IsNullOrEmpty(currentTimeZoneId))
         {
-            _clock = clock;
-            _distributedCache = distributedCache;
-            _httpContextAccessor = httpContextAccessor;
-            _userManager = userManager;
+            return null;
         }
 
-        public async Task<ITimeZone> GetUserTimeZoneAsync()
-        {
-            var currentTimeZoneId = await GetCurrentUserTimeZoneIdAsync();
-            if (string.IsNullOrEmpty(currentTimeZoneId))
-            {
-                return null;
-            }
+        return _clock.GetTimeZone(currentTimeZoneId);
+    }
 
-            return _clock.GetTimeZone(currentTimeZoneId);
-        }
+    public Task UpdateUserTimeZoneAsync(User user)
```

Comment:
I know this didn't change, but the method's name is misleading, since it actually only removes the cache entry. So e.g. `ClearUserTimeZoneCacheAsync`?